book-a-reading-room-visit.web/Views/Shared/AboutTheBookAReadingRoomVisitService.cshtml

```diff
@@ -38,8 +38,8 @@
                 <h2>Terms of booking</h2>
                 <p>New slots for booking will be made available every day.</p>
                 <p>Each booking is for one person only on a first-come, first-served basis – you will not be able to bring anyone with you, unless they book a visit themselves or have a duty of care.</p>
-                <p>There is a reasonable limit of three bookings of any type (standard or bulk) in a rolling 4-week period. Three bookings cannot be made for the same date.</p>
-                <p>Please be thoughtful and considerate of others when you book – we may cancel your booking if you exceed our reasonable limit of three bookings in 4 weeks.</p>
```

Comment:
Mixture of numeral and words for numbers again.

Reply:
These are editorial change given by editors